symfony-pack/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM quay.io/continuouspipe/symfony-php7.1-nginx:latest
+
+ARG NODE_VERSION=7.x
+ARG INSTALL_COMMON_PACKAGES=true
+ARG PHP_VERSION=7.1
+
+# Install Node.js
+RUN curl -sL "https://deb.nodesource.com/setup_$NODE_VERSION" > /tmp/install-node.sh \
+ && bash /tmp/install-node.sh \
+ && apt-get update -qq \
+ && DEBIAN_FRONTEND=noninteractive apt-get -qq -y --no-install-recommends install \
+    nodejs \
+ \
+ # Set up common NPM packages \
+ && if [ "$INSTALL_COMMON_PACKAGES" = "true" ]; then \
+      npm config set --global loglevel warn \
+   && npm install --global \
+      marked \
+      node-gyp \
+      gulp \
+      yarn \
+      npx \
+   # Install node-sass's linux bindings \
+   && npm rebuild node-sass; \
+    fi \
+ \
+ # Install PHP extensions \
+ && apt-get install -y php-amqp \
```

Comment:
argh awful awful amqp implementation. No support for multiple consumer callbacks :)

Reply:
do you know what uses this? It's better to use a amqp php library in my opinion

Reply:
> No support for multiple consumer callbacks :)

Erm, what do you mean?

> do you know what uses this? It's better to use a amqp php library in my opinion

I do on some projects. And soon Symfony is likely to use it in core. They've done a lot of changes recently if it  helps you to feel better 😛

Reply:
It's completely awful that it can't support consuming more than one queue at a time. Then you have to have multiple workers one each for each message type being processed, when as a multiplexed protocol it's multiple consumer capability can save resources when not fully utilised

Reply:
Whats worse is all their recent improvements aren't documented either

Reply:
I've looked at the source and tests, and it's still the case, it'd need to consume the channel not just the queue